components/edit-collective/Menu.js

```diff
@@ -216,7 +216,7 @@ const sectionsDisplayConditions = {
   [EDIT_COLLECTIVE_SECTIONS.PENDING_ORDERS]: () => false,
   [EDIT_COLLECTIVE_SECTIONS.SENDING_MONEY]: () => false,
   [EDIT_COLLECTIVE_SECTIONS.HOST_TWO_FACTOR_AUTH]: () => false,
-  [EDIT_COLLECTIVE_SECTIONS.HOST_VIRTUAL_CARDS]: () => false,
+  [EDIT_COLLECTIVE_SECTIONS.HOST_VIRTUAL_CARDS]: () => true,
```

Comment:
Seems out of scope

Reply:
Ops, this is a mistake. I was checking something on the virtual cards page and forgot to revert it. sorry about that.